Clinical-Genomics/cgweb
# removed validation of source

How to test:
0. install on stage
1. login in OP
2. chose type rml + any cust + any ordername 
3. add sample
4. enter valid information in all sample fields
5. press save sample
Expected outcome:
The sample form "collapses" to show that the sample has been saved
